### ISSUE TITLE
Better Marketing

### ISSUE THREAD

**Issue by boulter** [HubSpot/jinjava#165]
It would be nice to have a proper site with more extensive documentation, samples and community. 

See http://jtwig.org/ as an example.

**Comment by prior** [comment]
and for those docs -- would be nice to note the differences between jinja and jinjava --- like `.items()` vs `.iteritems()`

**Comment by boulter** [comment]
We should just fix that :)

**Comment by prior** [comment]
> We should just fix that :)

Agreed -- I went ahead and created an issue for that: https://github.com/HubSpot/jinjava/issues/167.

However, I believe my larger point is still valid.  There are apparently purposeful differences between jinja2 and jinjava where documentation would definitely be helpful.  For instance the presence of [`md5`](https://github.com/HubSpot/jinjava/blob/master/src/main/java/com/hubspot/jinjava/lib/filter/Md5Filter.java) and [`unixtimestamp`](https://github.com/HubSpot/jinjava/blob/master/src/main/java/com/hubspot/jinjava/lib/filter/UnixTimestampFilter.java) filters that are not part of the standard [jinja2 builtins](http://jinja.pocoo.org/docs/2.10/templates/#list-of-builtin-filters)

**Comment by boulter** [comment]
Maybe use https://docusaurus.io/ ?

**Comment by prior** [comment]
ooooo -- i really like that -- gonna have to remember docusaurus

**Comment by TheWebTech** [comment]
Us third-party HS developers would appreciate an official HS documentation site like this and I think at-least a handful of us that frequent the slack channel would help contribute.

(I'm Jon McLaren, from the HubSpot Design & Developers Slack)

**Comment by boulter** [comment]
Thanks, Jon. Any documentation here would be a subset of what's on designers.hubspot.com since there are HubSpot-specific extensions on top of Jinjava.

**Comment by TheWebTech** [comment]
I guess to clarify. I would actually support an open source project or some form of wiki for the HubSpot designer/developer documentation. It could link to the jinjava specific documentation wherever necessary to prevent accidental overlap.

**Comment by mattcoley** [comment]
Why not power open-source documentation with gitbooks and keep documentation in this repo similar to [Singularity](https://github.com/HubSpot/Singularity/tree/master/Docs)?

**Comment by boulter** [comment]
For any solution, I think the doc source should be in this repo.
